x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_search_preview/field_value_cell.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { css } from '@emotion/react';
+
+import { useEuiTheme, transparentize } from '@elastic/eui';
+
+import { FieldValue, isFieldValue } from './convert_results';
+
+export interface FieldValueCellProps {
+  value: FieldValue | string | number | boolean | null;
+}
+
+export const FieldValueCell: React.FC<FieldValueCellProps> = ({ value }) => {
+  const { euiTheme, colorMode } = useEuiTheme();
+
+  if (isFieldValue(value)) {
+    if (value.snippet) {
+      const [snippet] = value.snippet;
+      // https://github.com/elastic/eui/blob/c4afd758e4c42ba6b12ea58ccfd6486d5f544afb/src/components/mark/mark.styles.ts#L29-L36
```

Comment:
we can't get directly to `mark.styles.ts` because it isn't exported by eui. and using `EuiMark` directly would require us to parse the markup returned from elasticsearch and replace the `<em>` elements with `<EuiMark>`.

i tinkered around for a while on the parsing option but it felt really heavy for what ended up being four lines of copied code.